cmd/nvidia-ctk/hook/hook.go

```diff
@@ -41,6 +41,9 @@ func (m hookCommand) build() *cli.Command {
 	hook := cli.Command{
 		Name:  "hook",
```

Comment:
From the CLI libraries perspective -- does this define a subcommand named "hook"?

Reply:
if so, the code comment maybe should be "create 'hook' subcommand" (just noting this as I try to understand the code here)

Reply:
The CLI library that we use (urfave) has the following heirarchy:
A top-level CLI `App` can have one or more `Command`s. Each `Command` can have one or more `Subcommands`. 

Note that The `Subcommands` are themselves `Commands` which actually allows for arbitrary nesting.

I agree that from a documentation perspective, what we mean when we say "command" is "subcommand" and should use "command" for "app" (the top-level CLI) instead.